users/views.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth.tokens import default_token_generator
+from rest_framework import status, viewsets
+from rest_framework.decorators import action, api_view
+from rest_framework.response import Response
+from rest_framework.reverse import reverse
+
+from users.models import User
+from users.serializers import UserSerializer
+
+
+@api_view(['GET'])
+def api_root(request, format=None):
+    return Response({
+        'users': reverse('users:user_list', request=request, format=format),
+    })
+
+
+class UserViewSet(viewsets.ModelViewSet):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
```

Comment:
You created a serializer, but where do You use it?

Reply:
https://www.django-rest-framework.org/api-guide/serializers/

please read this documentation